README.md

```diff
@@ -42,4 +42,6 @@ Best to do a proper PR, rather than releasing from master, as missing one of the
 - Build with `bundle install`
 - Commit the `Gemfile.lock`
 - Merge upstream, do a Git tag and Git release
+- Pull from master and `bundle install`
```

Comment:
Do we need to run `bundle install` twice?

Reply:
Possibly this is redundant, though arguably it's good practice to reinstall after pulling from master?